_data/governments.yml

```diff
@@ -1059,6 +1059,7 @@ U.S. States:
 U.S. Tribal Nations:
   - coquille-indian-tribe
   - osage-nation
+  - MSBandofChoctawIndians
```

Comment:
Thanks for contribution! Could you maybe insert you entry in alphabetical order?

Reply:
I've submitted a new commit (d928ec7c8b9ddb135de72167140f363ae7ee0ffa) with the name in alphabetical order.